Testnet/DAOContract/DAOContract/DAOContract.cs

```diff
@@ -10,6 +10,12 @@ public Address Owner
         private set => State.SetAddress(nameof(Owner), value);
     }
 
+    private Address ClaimedOwner
+    {
+        get => State.GetAddress(nameof(ClaimedOwner));
+        set => State.SetAddress(nameof(ClaimedOwner), value);
+    }
```

Comment:
_should_ this be public with a private set?

Reply:
Both way is fine but i made it public and made setter private.